content/post/Example1.md

```diff
@@ -1,8 +1,42 @@
 ---
-title: "Mercurius ;>"
-date: 2023-10-21T14:00:00+05:30
+title: "Events by OSDC :-"
+date: 2023-11-14
 ---
 
 
 
-Mercurius test drive fellas, let's see how soon can we get it done!!!
\ No newline at end of file
+## Introduction:-
+
+Welcome to the Mercurius the official Newsletter of OSDC! We are going to explore the exciting events hosted by our community throughout the year and explore the rich history of our participation in programs like Google Summer of Code (GSOC), ASOC and other initiatives.
+
+## Events Round the Year:-
+
+Our community is vibrant and active, hosting a variety of events that serve to different interests. From workshops and hackathons to meetups and conferences, Our aim is to create engaging opportunities for learning and collaborations in OSS. we'll highlight some of the standout events that have shaped our community.
+
+### CTF (Unmasking the world of cybersecurity challenges)
+We demystify the practices of cybersecurity and go exploring into the labyrinth of Capture The Flag (CTF) challenges.we introduce you to the world of digital security and learn how to develop our skills by the help of CTFs. When it comes to cybersecurity, most of us are beginners and are intimidated by the intricacies of technology so this week we aim to help you by guiding you into the realms of cybersecurity.
```

Comment:
punctuation error ".we"

Reply:
use past tense (we're trying to present our work round the year which we do not trying to gather audience for a future event through this issue)